pkgs/nix-tools/darwin-rebuild.sh

```diff
@@ -255,7 +255,7 @@ if [ "$action" = changelog ]; then
   echo >&2
   echo "[1;1mCHANGELOG[0m" >&2
   echo >&2
-  head -n 32 "$systemConfig/darwin-changes"
+  "${PAGER:-less}" -- "$systemConfig/darwin-changes"
   echo >&2
```

Comment:
The pager eats up these extra lines in practice, so this could just be the pager line I think.

Unfortunately the pedantic shell quoting (which I generally approve of!) is counterproductive here – I have `$PAGER` set to `less -R`, which breaks with this.

Reply:
That's a good point I hadn't really paid attention to those `echo`. And you're right, if `PAGER` contains shell words as opposed to a single word resolving to an alias, we shouldn't quote it 👍